pkg/kubelet/volume/cache/actual_state_of_world.go

```diff
@@ -0,0 +1,586 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Package cache implements data structures used by the kubelet volume manager to
+keep track of attached volumes and the pods that mounted them.
+*/
+package cache
```

Comment:
I always found it annoying to have to look in another file to find the package comment. I realize that if a package has more than one file that the comment would be duplicated but I prefer that over a file no one ever looks at.

That said, let me know if you feel strongly and I can switch it.